js/utils/currency.js

```diff
@@ -153,7 +153,7 @@ export function formatCurrency(amount, currency, options) {
     // out with the appropriate Bitcoin symbol
     const formattedAmount = new Intl.NumberFormat(opts.locale, {
       minimumFractionDigits: 0,
-      maximumFractionDigits: 8,
+      maximumFractionDigits: 6,
```

Comment:
I think using 8 decimal places is problematic in a lot of layouts, especially tabular ones with columns with limited space. @morebrownies was even suggesting going down to 4 (that's what a lot of the designs have). I think 4 is little too low, so I though 6 could be a happy medium.

Another alternative, is leaving the default as 8 and then for certain situations were space is at a premium, passing in a smaller value. But... wouldn't we want to be consistent across the app? (plus the vast majority of places it's used in the UI have space constraints)